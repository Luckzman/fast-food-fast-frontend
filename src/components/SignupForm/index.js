import React from 'react';
import { Button, Checkbox, Form } from 'semantic-ui-react';

const SignupForm = () => (
  <Form>
    <Form.Group widths="equal">
      <Form.Input fluid placeholder="First name" />
      <Form.Input fluid placeholder="Last name" />
    </Form.Group>
    <Form.Field>
      <input placeholder="Email" />
    </Form.Field>
    <Form.Field>
      <input placeholder="Phone" />
    </Form.Field>
    <Form.Group widths="equal">
      <Form.Input type="password" fluid placeholder="Password" />
      <Form.Input type="password" fluid placeholder="Repeat Password" />
    </Form.Group>
    <Form.Field>
      <Checkbox label="I agree to the Terms and Conditions" />
    </Form.Field>
    <Button type="submit">Submit</Button>
  </Form>
);

export default SignupForm;
