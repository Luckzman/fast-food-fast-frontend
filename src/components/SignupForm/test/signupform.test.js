import React from 'react';
import { shallow } from 'enzyme';
import SignupForm from '..';

describe('App', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<SignupForm />);
    expect(wrapper).toMatchSnapshot();
  });
});
