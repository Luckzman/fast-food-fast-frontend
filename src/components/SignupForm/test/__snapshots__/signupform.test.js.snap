// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<form
  onSubmit={[Function]}
>
  <div
    className="form-input"
  >
    <Input
      className="name-input"
      handleChange={[Function]}
      name="firstname"
      placeholder="First name"
      type="text"
      value=""
    />
    <Input
      className="name-input"
      handleChange={[Function]}
      name="lastname"
      placeholder="Last name"
      type="text"
      value=""
    />
  </div>
  <Input
    className="email-input"
    handleChange={[Function]}
    name="email"
    placeholder="email"
    type="text"
    value=""
  />
  <Input
    className="email-input"
    handleChange={[Function]}
    name="phone"
    placeholder="phone"
    type="phone"
    value=""
  />
  <div
    className="form-input password-input"
  >
    <Input
      className="name-input"
      handleChange={[Function]}
      name="password"
      placeholder="password"
      type="password"
      value=""
    />
    <Input
      className="name-input"
      handleChange={[Function]}
      name="password2"
      placeholder="repeat password"
      type="password"
      value=""
    />
  </div>
  <Button
    className=""
    classes=""
    name="Submit"
    onClick={[Function]}
    type="submit"
  />
</form>
`;
